ui/src/pages/createAccount.tsx

```
@@ -102,6 +101,29 @@ export default function CreateAccountPage() {
               <option value="editor">Editor</option>
               <option value="viewer">Viewer</option>
             </select>
+            {role() && <h4 class="mt-2">Permission:</h4>}
+            {role() &&
+              roleDescriptions[role() as Role].map(
+                (desc: string, index: number) => (
+                  <li class="flex items-center space-x-3 text-white p-2 rounded-md transition-colors">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)